src/scss/components/switch/mixins/_hxSwitchFacade.scss

```diff
@@ -0,0 +1,61 @@
+//
+// ===== Mixins for <hx-checkbox> facade states =====
```

Comment:
Please update comment to reference `<hx-switch>` states.

Reply:
As facade mixins are not used , i removed this file